src/main/java/com/apeshko/javaspring/facade/impl/BookingFacadeImpl.java

```diff
@@ -0,0 +1,113 @@
+package com.apeshko.javaspring.facade.impl;
+
+
+import com.apeshko.javaspring.facade.BookingFacade;
+import com.apeshko.javaspring.model.Event;
+import com.apeshko.javaspring.model.Ticket;
+import com.apeshko.javaspring.model.User;
+import com.apeshko.javaspring.service.EventService;
+import com.apeshko.javaspring.service.TicketService;
+import com.apeshko.javaspring.service.UserService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+import java.util.List;
+
+@Component
```

Comment:
Seems it more service than component, don't forget about stereotype annotations ;)